week-4/implement/count.test.js

```diff
@@ -15,3 +17,34 @@
 // And a character char that does not exist within the case-sensitive str,
 // When the function is called with these inputs,
 // Then it should return 0, indicating that no occurrences of the char were found in the case-sensitive str.
+
+/////....................................................................................................
+//ANSWER
+
+// count.js
+function countChar(str, char) {
+  let count = 0;
+  for (let i = 0; i < str.length; i++) {
+    if (str[i] === char) {
+      count++;
+      while (str[i + 1] === char) {
+        i++;
+        count++;
+      }
+    }
+  }
+
+  return count;
+}
+
+
+test('counts occurrences of a character in a string', () => {
+  const inputString = "aaaaaaa";
+  const targetChar = "a";
+  const currentOutput = countChar(inputString, targetChar);
```

Comment:
You true master of readable code and & know the meaning of readable code. Well done @fhkahin